### ISSUE TITLE
Current FSLeyes install instructions update h5py, causing "AttributeError: 'str' object has no attribute 'decode'"

### ISSUE THREAD

**Issue by joshuacwnewton** [neuropoly/spinalcordtoolbox#3262]
Our [current documentation](https://spinalcordtoolbox.com/en/stable/user_section/installation.html#integration-with-fsleyes) recommends `conda install -y -c conda-forge fsleyes`.

<details>
<summary> Full FSLeyes install log on Ubuntu 20.04 </summary>

```console
> conda config --set channel_priority strict
> conda install -y -c conda-forge fsleyes
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): | WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2                                                                                                                                                                                  - WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.6.*, but conda is ignoring the .* and treating it as 3.6                                                                                                                                                                                                                             \ WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.*, but conda is ignoring the .* and treating it as 1                                                                                                                                                                                                                                 done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: done

## Package Plan ##

  environment location: /home/joshua/repos/spinalcordtoolbox/python/envs/venv_sct

  added / updated specs:
    - fsleyes


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    _libgcc_mutex-0.1          |      conda_forge           3 KB  conda-forge
    _openmp_mutex-4.5          |            1_gnu          22 KB  conda-forge
    alabaster-0.7.12           |             py_0          15 KB  conda-forge
    argon2-cffi-20.1.0         |   py36h8f6f2f9_2          47 KB  conda-forge
    async_generator-1.10       |             py_0          18 KB  conda-forge
    atk-1.0-2.36.0             |       h3371d22_4         560 KB  conda-forge
    attrs-20.3.0               |     pyhd3deb0d_0          41 KB  conda-forge
    babel-2.9.0                |     pyhd3deb0d_0         6.2 MB  conda-forge
    backcall-0.2.0             |     pyh9f0ad1d_0          13 KB  conda-forge
    backports-1.0              |             py_2           4 KB  conda-forge
    backports.functools_lru_cache-1.6.1|             py_0           8 KB  conda-forge
    bleach-3.3.0               |     pyh44b312d_0         111 KB  conda-forge
    brotlipy-0.7.0             |py36h8f6f2f9_1001         341 KB  conda-forge
    c-ares-1.17.1              |       h36c2ea0_0         111 KB  conda-forge
    ca-certificates-2020.12.5  |       ha878542_0         137 KB  conda-forge
    cached-property-1.5.1      |             py_0          10 KB  conda-forge
    cairo-1.16.0               |    h7979940_1007         1.5 MB  conda-forge
    certifi-2020.12.5          |   py36h5fab9bb_1         143 KB  conda-forge
    cffi-1.14.5                |   py36hc120d54_0         225 KB  conda-forge
    chardet-4.0.0              |   py36h5fab9bb_1         219 KB  conda-forge
    colorama-0.4.4             |     pyh9f0ad1d_0          18 KB  conda-forge
    cryptography-3.4.4         |   py36hc39840e_0         631 KB  conda-forge
    cycler-0.10.0              |             py_2           9 KB  conda-forge
    dataclasses-0.7            |     pyhe4b4509_6          21 KB  conda-forge
    dcm2niix-1.0.20181114      |    h6bb024c_1000         168 KB  conda-forge
    decorator-4.4.2            |             py_0          11 KB  conda-forge
    defusedxml-0.6.0           |             py_0          22 KB  conda-forge
    docutils-0.16              |   py36h5fab9bb_3         738 KB  conda-forge
    entrypoints-0.3            |  pyhd8ed1ab_1003           8 KB  conda-forge
    expat-2.2.10               |       h9c3ff4c_0         164 KB  conda-forge
    fontconfig-2.13.1          |    hba837de_1004         344 KB  conda-forge
    freeglut-3.0.0             |       hfc679d8_5         247 KB  conda-forge
    freetype-2.10.4            |       h0708190_1         890 KB  conda-forge
    fribidi-1.0.10             |       h36c2ea0_0         112 KB  conda-forge
    fsleyes-0.32.3             |           py36_0        28.2 MB  conda-forge
    fsleyes-props-1.7.0        |             py_0          90 KB  conda-forge
    fsleyes-widgets-0.11.0     |     pyhd8ed1ab_0         505 KB  conda-forge
    fslpy-3.5.3                |     pyh050c7b8_0         3.6 MB  conda-forge
    gdk-pixbuf-2.42.2          |       h0c95a7a_2         611 KB  conda-forge
    gettext-0.19.8.1           |    h0b5b191_1005         3.6 MB  conda-forge
    glib-2.66.7                |       h9c3ff4c_1         443 KB  conda-forge
    glib-tools-2.66.7          |       h9c3ff4c_1          85 KB  conda-forge
    graphite2-1.3.13           |    h58526e2_1001         102 KB  conda-forge
    gst-plugins-base-1.18.3    |       h04508c2_0         2.5 MB  conda-forge
    gstreamer-1.18.3           |       h3560a44_0         2.0 MB  conda-forge
    gtk2-2.24.33               |       hab0c2f8_0         7.3 MB  conda-forge
    h5py-3.1.0                 |nompi_py36hc1bc4f5_100         1.2 MB  conda-forge
    harfbuzz-2.7.4             |       h5cf4720_0         1.9 MB  conda-forge
    hdf5-1.10.6                |nompi_h6a2412b_1114         3.1 MB  conda-forge
    icu-68.1                   |       h58526e2_0        13.0 MB  conda-forge
    idna-2.10                  |     pyh9f0ad1d_0          52 KB  conda-forge
    imagesize-1.2.0            |             py_0           8 KB  conda-forge
    importlib-metadata-3.7.0   |   py36h5fab9bb_0          23 KB  conda-forge
    indexed_gzip-1.4.0         |   py36h575eb5e_0         523 KB  conda-forge
    ipykernel-5.5.0            |   py36he448a4c_1         166 KB  conda-forge
    ipython-7.16.1             |   py36he448a4c_2         1.1 MB  conda-forge
    ipython_genutils-0.2.0     |             py_1          21 KB  conda-forge
    isodate-0.6.0              |             py_1          25 KB  conda-forge
    jedi-0.17.2                |   py36h5fab9bb_1         957 KB  conda-forge
    jinja2-2.11.3              |     pyh44b312d_0          93 KB  conda-forge
    jpeg-9d                    |       h36c2ea0_0         264 KB  conda-forge
    jsonschema-3.2.0           |     pyhd8ed1ab_3          45 KB  conda-forge
    jupyter_client-6.1.11      |     pyhd8ed1ab_1          76 KB  conda-forge
    jupyter_core-4.7.1         |   py36h5fab9bb_0          72 KB  conda-forge
    jupyterlab_pygments-0.1.2  |     pyh9f0ad1d_0           8 KB  conda-forge
    kiwisolver-1.3.1           |   py36h605e78d_1          78 KB  conda-forge
    krb5-1.17.2                |       h926e7f8_0         1.4 MB  conda-forge
    lcms2-2.12                 |       hddcbb42_0         443 KB  conda-forge
    ld_impl_linux-64-2.35.1    |       hea4e1c9_2         618 KB  conda-forge
    libblas-3.9.0              |       8_openblas          11 KB  conda-forge
    libcblas-3.9.0             |       8_openblas          11 KB  conda-forge
    libcurl-7.71.1             |       hcdd3856_8         312 KB  conda-forge
    libedit-3.1.20191231       |       he28a2e2_2         121 KB  conda-forge
    libev-4.33                 |       h516909a_1         104 KB  conda-forge
    libffi-3.3                 |       h58526e2_2          51 KB  conda-forge
    libgcc-ng-9.3.0            |      h2828fa1_18         7.8 MB  conda-forge
    libgfortran-ng-9.3.0       |      hff62375_18          22 KB  conda-forge
    libgfortran5-9.3.0         |      hff62375_18         2.0 MB  conda-forge
    libglib-2.66.7             |       h3e27bee_1         3.0 MB  conda-forge
    libglu-9.0.0               |    he1b5a44_1001         413 KB  conda-forge
    libgomp-9.3.0              |      h2828fa1_18         376 KB  conda-forge
    libiconv-1.16              |       h516909a_0         1.4 MB  conda-forge
    liblapack-3.9.0            |       8_openblas          11 KB  conda-forge
    libnghttp2-1.43.0          |       h812cca2_0         808 KB  conda-forge
    libopenblas-0.3.12         |pthreads_h4812303_1         8.9 MB  conda-forge
    libpng-1.6.37              |       h21135ba_2         306 KB  conda-forge
    libsodium-1.0.18           |       h36c2ea0_1         366 KB  conda-forge
    libspatialindex-1.9.3      |       h9c3ff4c_3         4.6 MB  conda-forge
    libssh2-1.9.0              |       hab1572f_5         225 KB  conda-forge
    libstdcxx-ng-9.3.0         |      h6de172a_18         4.0 MB  conda-forge
    libtiff-4.2.0              |       hdc55705_0         633 KB  conda-forge
    libuuid-2.32.1             |    h7f98852_1000          28 KB  conda-forge
    libwebp-base-1.2.0         |       h7f98852_0         808 KB  conda-forge
    libxcb-1.13                |    h7f98852_1003         395 KB  conda-forge
    libxml2-2.9.10             |       h72842e0_3         1.3 MB  conda-forge
    lz4-c-1.9.3                |       h9c3ff4c_0         179 KB  conda-forge
    markupsafe-1.1.1           |   py36h8f6f2f9_3          27 KB  conda-forge
    matplotlib-base-3.3.4      |   py36hd391965_0         6.8 MB  conda-forge
    mistune-0.8.4              |py36h8f6f2f9_1003          54 KB  conda-forge
    nbclient-0.5.3             |     pyhd8ed1ab_0          67 KB  conda-forge
    nbconvert-6.0.7            |   py36h5fab9bb_3         533 KB  conda-forge
    nbformat-5.1.2             |     pyhd8ed1ab_1          66 KB  conda-forge
    ncurses-6.2                |       h58526e2_4         985 KB  conda-forge
    nest-asyncio-1.4.3         |     pyhd8ed1ab_0           9 KB  conda-forge
    nibabel-3.2.1              |     pyhd8ed1ab_0         2.7 MB  conda-forge
    notebook-6.2.0             |   py36h5fab9bb_0         6.2 MB  conda-forge
    numpy-1.19.5               |   py36h2aa4a07_1         5.3 MB  conda-forge
    olefile-0.46               |     pyh9f0ad1d_1          32 KB  conda-forge
    openssl-1.1.1j             |       h7f98852_0         2.1 MB  conda-forge
    packaging-20.9             |     pyh44b312d_0          35 KB  conda-forge
    pandoc-2.11.4              |       h7f98852_0        17.9 MB  conda-forge
    pandocfilters-1.4.2        |             py_1           9 KB  conda-forge
    pango-1.42.4               |       h69149e4_5         533 KB  conda-forge
    parso-0.7.1                |     pyh9f0ad1d_0          70 KB  conda-forge
    pathlib2-2.3.5             |   py36h5fab9bb_3          35 KB  conda-forge
    pcre-8.44                  |       he1b5a44_0         261 KB  conda-forge
    pexpect-4.8.0              |     pyh9f0ad1d_2          47 KB  conda-forge
    pickleshare-0.7.5          |          py_1003           9 KB  conda-forge
    pillow-8.1.1               |   py36ha6010c0_0         679 KB  conda-forge
    pip-21.0.1                 |     pyhd8ed1ab_0         1.1 MB  conda-forge
    pixman-0.40.0              |       h36c2ea0_0         627 KB  conda-forge
    progressbar2-3.53.1        |     pyh9f0ad1d_0          25 KB  conda-forge
    prometheus_client-0.9.0    |     pyhd3deb0d_0          44 KB  conda-forge
    prompt-toolkit-3.0.16      |     pyha770c72_0         244 KB  conda-forge
    pthread-stubs-0.4          |    h36c2ea0_1001           5 KB  conda-forge
    ptyprocess-0.7.0           |     pyhd3deb0d_0          16 KB  conda-forge
    pycparser-2.20             |     pyh9f0ad1d_2          94 KB  conda-forge
    pydicom-2.1.2              |     pyhd3deb0d_0         1.3 MB  conda-forge
    pygments-2.8.0             |     pyhd8ed1ab_0         736 KB  conda-forge
    pyopengl-3.1.5             |             py_0         854 KB  conda-forge
    pyopenssl-20.0.1           |     pyhd8ed1ab_0          48 KB  conda-forge
    pyparsing-2.4.7            |     pyh9f0ad1d_0          60 KB  conda-forge
    pypubsub-4.0.3             |             py_0          44 KB  conda-forge
    pyrsistent-0.17.3          |   py36h8f6f2f9_2          89 KB  conda-forge
    pysocks-1.7.1              |   py36h5fab9bb_3          27 KB  conda-forge
    python-3.6.13              |hffdb5ce_0_cpython        38.4 MB  conda-forge
    python-dateutil-2.8.1      |             py_0         220 KB  conda-forge
    python-utils-2.5.5         |     pyh44b312d_0          15 KB  conda-forge
    python_abi-3.6             |          1_cp36m           4 KB  conda-forge
    pytz-2021.1                |     pyhd8ed1ab_0         239 KB  conda-forge
    pyzmq-22.0.3               |   py36h7068817_1         496 KB  conda-forge
    readline-8.0               |       he28a2e2_2         281 KB  conda-forge
    requests-2.25.1            |     pyhd3deb0d_0          51 KB  conda-forge
    rtree-0.9.7                |   py36hb415506_1          45 KB  conda-forge
    scipy-1.5.3                |   py36h9e8f40b_0        19.1 MB  conda-forge
    send2trash-1.5.0           |             py_0          12 KB  conda-forge
    setuptools-49.6.0          |   py36h5fab9bb_3         936 KB  conda-forge
    six-1.15.0                 |     pyh9f0ad1d_0          14 KB  conda-forge
    snowballstemmer-2.1.0      |     pyhd8ed1ab_0          57 KB  conda-forge
    sphinx-3.5.1               |     pyhd8ed1ab_0         1.5 MB  conda-forge
    sphinx_rtd_theme-0.5.1     |     pyhd3deb0d_0         2.7 MB  conda-forge
    sphinxcontrib-applehelp-1.0.2|             py_0          28 KB  conda-forge
    sphinxcontrib-devhelp-1.0.2|             py_0          22 KB  conda-forge
    sphinxcontrib-htmlhelp-1.0.3|             py_0          27 KB  conda-forge
    sphinxcontrib-jsmath-1.0.1 |             py_0           7 KB  conda-forge
    sphinxcontrib-qthelp-1.0.3 |             py_0          25 KB  conda-forge
    sphinxcontrib-serializinghtml-1.1.4|             py_0          24 KB  conda-forge
    sqlite-3.34.0              |       h74cdb3f_0         1.4 MB  conda-forge
    terminado-0.9.2            |   py36h5fab9bb_0          26 KB  conda-forge
    testpath-0.4.4             |             py_0          85 KB  conda-forge
    tk-8.6.10                  |       h21135ba_1         3.2 MB  conda-forge
    tornado-6.1                |   py36h8f6f2f9_1         643 KB  conda-forge
    traitlets-4.3.3            |   py36h9f0ad1d_1         133 KB  conda-forge
    trimesh-3.9.8              |     pyh44b312d_0         504 KB  conda-forge
    typing_extensions-3.7.4.3  |             py_0          25 KB  conda-forge
    urllib3-1.26.3             |     pyhd8ed1ab_0          99 KB  conda-forge
    wcwidth-0.2.5              |     pyh9f0ad1d_2          33 KB  conda-forge
    webencodings-0.5.1         |             py_1          12 KB  conda-forge
    wheel-0.36.2               |     pyhd3deb0d_0          31 KB  conda-forge
    wxnatpy-0.3.2              |             py_0          19 KB  conda-forge
    wxpython-4.1.1             |   py36hdbe9d72_0        21.3 MB  conda-forge
    xnat-0.3.25                |     pyh9f0ad1d_0          48 KB  conda-forge
    xorg-kbproto-1.0.7         |    h7f98852_1002          27 KB  conda-forge
    xorg-libice-1.0.10         |       h7f98852_0          58 KB  conda-forge
    xorg-libsm-1.2.3           |    h84519dc_1000          25 KB  conda-forge
    xorg-libx11-1.7.0          |       h7f98852_0         923 KB  conda-forge
    xorg-libxau-1.0.9          |       h7f98852_0          13 KB  conda-forge
    xorg-libxdmcp-1.1.3        |       h7f98852_0          19 KB  conda-forge
    xorg-libxext-1.3.4         |       h7f98852_1          54 KB  conda-forge
    xorg-libxrender-0.9.10     |    h7f98852_1003          32 KB  conda-forge
    xorg-renderproto-0.11.1    |    h7f98852_1002           9 KB  conda-forge
    xorg-xextproto-7.3.0       |    h7f98852_1002          28 KB  conda-forge
    xorg-xproto-7.0.31         |    h7f98852_1007          73 KB  conda-forge
    xz-5.2.5                   |       h516909a_1         343 KB  conda-forge
    zeromq-4.3.4               |       h9c3ff4c_0         352 KB  conda-forge
    zipp-3.4.0                 |             py_0          11 KB  conda-forge
    zlib-1.2.11                |    h516909a_1010         106 KB  conda-forge
    zstd-1.4.9                 |       ha95c52a_0         431 KB  conda-forge
    ------------------------------------------------------------
                                           Total:       268.8 MB

The following NEW packages will be INSTALLED:

  _openmp_mutex      conda-forge/linux-64::_openmp_mutex-4.5-1_gnu
  alabaster          conda-forge/noarch::alabaster-0.7.12-py_0
  argon2-cffi        conda-forge/linux-64::argon2-cffi-20.1.0-py36h8f6f2f9_2
  async_generator    conda-forge/noarch::async_generator-1.10-py_0
  atk-1.0            conda-forge/linux-64::atk-1.0-2.36.0-h3371d22_4
  attrs              conda-forge/noarch::attrs-20.3.0-pyhd3deb0d_0
  babel              conda-forge/noarch::babel-2.9.0-pyhd3deb0d_0
  backcall           conda-forge/noarch::backcall-0.2.0-pyh9f0ad1d_0
  backports          conda-forge/noarch::backports-1.0-py_2
  backports.functoo~ conda-forge/noarch::backports.functools_lru_cache-1.6.1-py_0
  bleach             conda-forge/noarch::bleach-3.3.0-pyh44b312d_0
  brotlipy           conda-forge/linux-64::brotlipy-0.7.0-py36h8f6f2f9_1001
  c-ares             conda-forge/linux-64::c-ares-1.17.1-h36c2ea0_0
  cached-property    conda-forge/noarch::cached-property-1.5.1-py_0
  cairo              conda-forge/linux-64::cairo-1.16.0-h7979940_1007
  cffi               conda-forge/linux-64::cffi-1.14.5-py36hc120d54_0
  chardet            conda-forge/linux-64::chardet-4.0.0-py36h5fab9bb_1
  colorama           conda-forge/noarch::colorama-0.4.4-pyh9f0ad1d_0
  cryptography       conda-forge/linux-64::cryptography-3.4.4-py36hc39840e_0
  cycler             conda-forge/noarch::cycler-0.10.0-py_2
  dataclasses        conda-forge/noarch::dataclasses-0.7-pyhe4b4509_6
  dcm2niix           conda-forge/linux-64::dcm2niix-1.0.20181114-h6bb024c_1000
  decorator          conda-forge/noarch::decorator-4.4.2-py_0
  defusedxml         conda-forge/noarch::defusedxml-0.6.0-py_0
  docutils           conda-forge/linux-64::docutils-0.16-py36h5fab9bb_3
  entrypoints        conda-forge/noarch::entrypoints-0.3-pyhd8ed1ab_1003
  expat              conda-forge/linux-64::expat-2.2.10-h9c3ff4c_0
  fontconfig         conda-forge/linux-64::fontconfig-2.13.1-hba837de_1004
  freeglut           conda-forge/linux-64::freeglut-3.0.0-hfc679d8_5
  freetype           conda-forge/linux-64::freetype-2.10.4-h0708190_1
  fribidi            conda-forge/linux-64::fribidi-1.0.10-h36c2ea0_0
  fsleyes            conda-forge/linux-64::fsleyes-0.32.3-py36_0
  fsleyes-props      conda-forge/noarch::fsleyes-props-1.7.0-py_0
  fsleyes-widgets    conda-forge/noarch::fsleyes-widgets-0.11.0-pyhd8ed1ab_0
  fslpy              conda-forge/noarch::fslpy-3.5.3-pyh050c7b8_0
  gdk-pixbuf         conda-forge/linux-64::gdk-pixbuf-2.42.2-h0c95a7a_2
  gettext            conda-forge/linux-64::gettext-0.19.8.1-h0b5b191_1005
  glib               conda-forge/linux-64::glib-2.66.7-h9c3ff4c_1
  glib-tools         conda-forge/linux-64::glib-tools-2.66.7-h9c3ff4c_1
  graphite2          conda-forge/linux-64::graphite2-1.3.13-h58526e2_1001
  gst-plugins-base   conda-forge/linux-64::gst-plugins-base-1.18.3-h04508c2_0
  gstreamer          conda-forge/linux-64::gstreamer-1.18.3-h3560a44_0
  gtk2               conda-forge/linux-64::gtk2-2.24.33-hab0c2f8_0
  h5py               conda-forge/linux-64::h5py-3.1.0-nompi_py36hc1bc4f5_100
  harfbuzz           conda-forge/linux-64::harfbuzz-2.7.4-h5cf4720_0
  hdf5               conda-forge/linux-64::hdf5-1.10.6-nompi_h6a2412b_1114
  icu                conda-forge/linux-64::icu-68.1-h58526e2_0
  idna               conda-forge/noarch::idna-2.10-pyh9f0ad1d_0
  imagesize          conda-forge/noarch::imagesize-1.2.0-py_0
  importlib-metadata conda-forge/linux-64::importlib-metadata-3.7.0-py36h5fab9bb_0
  indexed_gzip       conda-forge/linux-64::indexed_gzip-1.4.0-py36h575eb5e_0
  ipykernel          conda-forge/linux-64::ipykernel-5.5.0-py36he448a4c_1
  ipython            conda-forge/linux-64::ipython-7.16.1-py36he448a4c_2
  ipython_genutils   conda-forge/noarch::ipython_genutils-0.2.0-py_1
  isodate            conda-forge/noarch::isodate-0.6.0-py_1
  jedi               conda-forge/linux-64::jedi-0.17.2-py36h5fab9bb_1
  jinja2             conda-forge/noarch::jinja2-2.11.3-pyh44b312d_0
  jpeg               conda-forge/linux-64::jpeg-9d-h36c2ea0_0
  jsonschema         conda-forge/noarch::jsonschema-3.2.0-pyhd8ed1ab_3
  jupyter_client     conda-forge/noarch::jupyter_client-6.1.11-pyhd8ed1ab_1
  jupyter_core       conda-forge/linux-64::jupyter_core-4.7.1-py36h5fab9bb_0
  jupyterlab_pygmen~ conda-forge/noarch::jupyterlab_pygments-0.1.2-pyh9f0ad1d_0
  kiwisolver         conda-forge/linux-64::kiwisolver-1.3.1-py36h605e78d_1
  krb5               conda-forge/linux-64::krb5-1.17.2-h926e7f8_0
  lcms2              conda-forge/linux-64::lcms2-2.12-hddcbb42_0
  libblas            conda-forge/linux-64::libblas-3.9.0-8_openblas
  libcblas           conda-forge/linux-64::libcblas-3.9.0-8_openblas
  libcurl            conda-forge/linux-64::libcurl-7.71.1-hcdd3856_8
  libev              conda-forge/linux-64::libev-4.33-h516909a_1
  libgfortran-ng     conda-forge/linux-64::libgfortran-ng-9.3.0-hff62375_18
  libgfortran5       conda-forge/linux-64::libgfortran5-9.3.0-hff62375_18
  libglib            conda-forge/linux-64::libglib-2.66.7-h3e27bee_1
  libglu             conda-forge/linux-64::libglu-9.0.0-he1b5a44_1001
  libgomp            conda-forge/linux-64::libgomp-9.3.0-h2828fa1_18
  libiconv           conda-forge/linux-64::libiconv-1.16-h516909a_0
  liblapack          conda-forge/linux-64::liblapack-3.9.0-8_openblas
  libnghttp2         conda-forge/linux-64::libnghttp2-1.43.0-h812cca2_0
  libopenblas        conda-forge/linux-64::libopenblas-0.3.12-pthreads_h4812303_1
  libpng             conda-forge/linux-64::libpng-1.6.37-h21135ba_2
  libsodium          conda-forge/linux-64::libsodium-1.0.18-h36c2ea0_1
  libspatialindex    conda-forge/linux-64::libspatialindex-1.9.3-h9c3ff4c_3
  libssh2            conda-forge/linux-64::libssh2-1.9.0-hab1572f_5
  libtiff            conda-forge/linux-64::libtiff-4.2.0-hdc55705_0
  libuuid            conda-forge/linux-64::libuuid-2.32.1-h7f98852_1000
  libwebp-base       conda-forge/linux-64::libwebp-base-1.2.0-h7f98852_0
  libxcb             conda-forge/linux-64::libxcb-1.13-h7f98852_1003
  libxml2            conda-forge/linux-64::libxml2-2.9.10-h72842e0_3
  lz4-c              conda-forge/linux-64::lz4-c-1.9.3-h9c3ff4c_0
  markupsafe         conda-forge/linux-64::markupsafe-1.1.1-py36h8f6f2f9_3
  matplotlib-base    conda-forge/linux-64::matplotlib-base-3.3.4-py36hd391965_0
  mistune            conda-forge/linux-64::mistune-0.8.4-py36h8f6f2f9_1003
  nbclient           conda-forge/noarch::nbclient-0.5.3-pyhd8ed1ab_0
  nbconvert          conda-forge/linux-64::nbconvert-6.0.7-py36h5fab9bb_3
  nbformat           conda-forge/noarch::nbformat-5.1.2-pyhd8ed1ab_1
  nest-asyncio       conda-forge/noarch::nest-asyncio-1.4.3-pyhd8ed1ab_0
  nibabel            conda-forge/noarch::nibabel-3.2.1-pyhd8ed1ab_0
  notebook           conda-forge/linux-64::notebook-6.2.0-py36h5fab9bb_0
  numpy              conda-forge/linux-64::numpy-1.19.5-py36h2aa4a07_1
  olefile            conda-forge/noarch::olefile-0.46-pyh9f0ad1d_1
  packaging          conda-forge/noarch::packaging-20.9-pyh44b312d_0
  pandoc             conda-forge/linux-64::pandoc-2.11.4-h7f98852_0
  pandocfilters      conda-forge/noarch::pandocfilters-1.4.2-py_1
  pango              conda-forge/linux-64::pango-1.42.4-h69149e4_5
  parso              conda-forge/noarch::parso-0.7.1-pyh9f0ad1d_0
  pathlib2           conda-forge/linux-64::pathlib2-2.3.5-py36h5fab9bb_3
  pcre               conda-forge/linux-64::pcre-8.44-he1b5a44_0
  pexpect            conda-forge/noarch::pexpect-4.8.0-pyh9f0ad1d_2
  pickleshare        conda-forge/noarch::pickleshare-0.7.5-py_1003
  pillow             conda-forge/linux-64::pillow-8.1.1-py36ha6010c0_0
  pixman             conda-forge/linux-64::pixman-0.40.0-h36c2ea0_0
  progressbar2       conda-forge/noarch::progressbar2-3.53.1-pyh9f0ad1d_0
  prometheus_client  conda-forge/noarch::prometheus_client-0.9.0-pyhd3deb0d_0
  prompt-toolkit     conda-forge/noarch::prompt-toolkit-3.0.16-pyha770c72_0
  pthread-stubs      conda-forge/linux-64::pthread-stubs-0.4-h36c2ea0_1001
  ptyprocess         conda-forge/noarch::ptyprocess-0.7.0-pyhd3deb0d_0
  pycparser          conda-forge/noarch::pycparser-2.20-pyh9f0ad1d_2
  pydicom            conda-forge/noarch::pydicom-2.1.2-pyhd3deb0d_0
  pygments           conda-forge/noarch::pygments-2.8.0-pyhd8ed1ab_0
  pyopengl           conda-forge/noarch::pyopengl-3.1.5-py_0
  pyopenssl          conda-forge/noarch::pyopenssl-20.0.1-pyhd8ed1ab_0
  pyparsing          conda-forge/noarch::pyparsing-2.4.7-pyh9f0ad1d_0
  pypubsub           conda-forge/noarch::pypubsub-4.0.3-py_0
  pyrsistent         conda-forge/linux-64::pyrsistent-0.17.3-py36h8f6f2f9_2
  pysocks            conda-forge/linux-64::pysocks-1.7.1-py36h5fab9bb_3
  python-dateutil    conda-forge/noarch::python-dateutil-2.8.1-py_0
  python-utils       conda-forge/noarch::python-utils-2.5.5-pyh44b312d_0
  python_abi         conda-forge/linux-64::python_abi-3.6-1_cp36m
  pytz               conda-forge/noarch::pytz-2021.1-pyhd8ed1ab_0
  pyzmq              conda-forge/linux-64::pyzmq-22.0.3-py36h7068817_1
  requests           conda-forge/noarch::requests-2.25.1-pyhd3deb0d_0
  rtree              conda-forge/linux-64::rtree-0.9.7-py36hb415506_1
  scipy              conda-forge/linux-64::scipy-1.5.3-py36h9e8f40b_0
  send2trash         conda-forge/noarch::send2trash-1.5.0-py_0
  six                conda-forge/noarch::six-1.15.0-pyh9f0ad1d_0
  snowballstemmer    conda-forge/noarch::snowballstemmer-2.1.0-pyhd8ed1ab_0
  sphinx             conda-forge/noarch::sphinx-3.5.1-pyhd8ed1ab_0
  sphinx_rtd_theme   conda-forge/noarch::sphinx_rtd_theme-0.5.1-pyhd3deb0d_0
  sphinxcontrib-app~ conda-forge/noarch::sphinxcontrib-applehelp-1.0.2-py_0
  sphinxcontrib-dev~ conda-forge/noarch::sphinxcontrib-devhelp-1.0.2-py_0
  sphinxcontrib-htm~ conda-forge/noarch::sphinxcontrib-htmlhelp-1.0.3-py_0
  sphinxcontrib-jsm~ conda-forge/noarch::sphinxcontrib-jsmath-1.0.1-py_0
  sphinxcontrib-qth~ conda-forge/noarch::sphinxcontrib-qthelp-1.0.3-py_0
  sphinxcontrib-ser~ conda-forge/noarch::sphinxcontrib-serializinghtml-1.1.4-py_0
  terminado          conda-forge/linux-64::terminado-0.9.2-py36h5fab9bb_0
  testpath           conda-forge/noarch::testpath-0.4.4-py_0
  tornado            conda-forge/linux-64::tornado-6.1-py36h8f6f2f9_1
  traitlets          conda-forge/linux-64::traitlets-4.3.3-py36h9f0ad1d_1
  trimesh            conda-forge/noarch::trimesh-3.9.8-pyh44b312d_0
  typing_extensions  conda-forge/noarch::typing_extensions-3.7.4.3-py_0
  urllib3            conda-forge/noarch::urllib3-1.26.3-pyhd8ed1ab_0
  wcwidth            conda-forge/noarch::wcwidth-0.2.5-pyh9f0ad1d_2
  webencodings       conda-forge/noarch::webencodings-0.5.1-py_1
  wxnatpy            conda-forge/noarch::wxnatpy-0.3.2-py_0
  wxpython           conda-forge/linux-64::wxpython-4.1.1-py36hdbe9d72_0
  xnat               conda-forge/noarch::xnat-0.3.25-pyh9f0ad1d_0
  xorg-kbproto       conda-forge/linux-64::xorg-kbproto-1.0.7-h7f98852_1002
  xorg-libice        conda-forge/linux-64::xorg-libice-1.0.10-h7f98852_0
  xorg-libsm         conda-forge/linux-64::xorg-libsm-1.2.3-h84519dc_1000
  xorg-libx11        conda-forge/linux-64::xorg-libx11-1.7.0-h7f98852_0
  xorg-libxau        conda-forge/linux-64::xorg-libxau-1.0.9-h7f98852_0
  xorg-libxdmcp      conda-forge/linux-64::xorg-libxdmcp-1.1.3-h7f98852_0
  xorg-libxext       conda-forge/linux-64::xorg-libxext-1.3.4-h7f98852_1
  xorg-libxrender    conda-forge/linux-64::xorg-libxrender-0.9.10-h7f98852_1003
  xorg-renderproto   conda-forge/linux-64::xorg-renderproto-0.11.1-h7f98852_1002
  xorg-xextproto     conda-forge/linux-64::xorg-xextproto-7.3.0-h7f98852_1002
  xorg-xproto        conda-forge/linux-64::xorg-xproto-7.0.31-h7f98852_1007
  zeromq             conda-forge/linux-64::zeromq-4.3.4-h9c3ff4c_0
  zipp               conda-forge/noarch::zipp-3.4.0-py_0
  zstd               conda-forge/linux-64::zstd-1.4.9-ha95c52a_0

The following packages will be UPDATED:

  certifi            pkgs/main::certifi-2020.12.5-py36h06a~ --> conda-forge::certifi-2020.12.5-py36h5fab9bb_1
  ld_impl_linux-64   pkgs/main::ld_impl_linux-64-2.33.1-h5~ --> conda-forge::ld_impl_linux-64-2.35.1-hea4e1c9_2
  libedit            pkgs/main::libedit-3.1.20191231-h14c3~ --> conda-forge::libedit-3.1.20191231-he28a2e2_2
  libgcc-ng           pkgs/main::libgcc-ng-9.1.0-hdf63c60_0 --> conda-forge::libgcc-ng-9.3.0-h2828fa1_18
  libstdcxx-ng       pkgs/main::libstdcxx-ng-9.1.0-hdf63c6~ --> conda-forge::libstdcxx-ng-9.3.0-h6de172a_18
  ncurses                 pkgs/main::ncurses-6.2-he6710b0_1 --> conda-forge::ncurses-6.2-h58526e2_4
  sqlite                pkgs/main::sqlite-3.33.0-h62c20be_0 --> conda-forge::sqlite-3.34.0-h74cdb3f_0
  tk                        pkgs/main::tk-8.6.10-hbc83047_0 --> conda-forge::tk-8.6.10-h21135ba_1
  xz                         pkgs/main::xz-5.2.5-h7b6447c_0 --> conda-forge::xz-5.2.5-h516909a_1
  zlib                    pkgs/main::zlib-1.2.11-h7b6447c_3 --> conda-forge::zlib-1.2.11-h516909a_1010

The following packages will be SUPERSEDED by a higher-priority channel:

  _libgcc_mutex           pkgs/main::_libgcc_mutex-0.1-main --> conda-forge::_libgcc_mutex-0.1-conda_forge
  ca-certificates    pkgs/main::ca-certificates-2021.1.19-~ --> conda-forge::ca-certificates-2020.12.5-ha878542_0
  libffi                   pkgs/main::libffi-3.3-he6710b0_2 --> conda-forge::libffi-3.3-h58526e2_2
  openssl              pkgs/main::openssl-1.1.1j-h27cfd23_0 --> conda-forge::openssl-1.1.1j-h7f98852_0
  pip                pkgs/main/linux-64::pip-21.0.1-py36h0~ --> conda-forge/noarch::pip-21.0.1-pyhd8ed1ab_0
  python                pkgs/main::python-3.6.13-hdb3f193_0 --> conda-forge::python-3.6.13-hffdb5ce_0_cpython
  readline               pkgs/main::readline-8.1-h27cfd23_0 --> conda-forge::readline-8.0-he28a2e2_2
  setuptools         pkgs/main::setuptools-52.0.0-py36h06a~ --> conda-forge::setuptools-49.6.0-py36h5fab9bb_3
  wheel                pkgs/main::wheel-0.36.2-pyhd3eb1b0_0 --> conda-forge::wheel-0.36.2-pyhd3deb0d_0



Downloading and Extracting Packages
pysocks-1.7.1        | 27 KB     | ######################################################################################################################################################################################### | 100% 
cairo-1.16.0         | 1.5 MB    | ######################################################################################################################################################################################### | 100% 
decorator-4.4.2      | 11 KB     | ######################################################################################################################################################################################### | 100% 
prompt-toolkit-3.0.1 | 244 KB    | ######################################################################################################################################################################################### | 100% 
fsleyes-props-1.7.0  | 90 KB     | ######################################################################################################################################################################################### | 100% 
krb5-1.17.2          | 1.4 MB    | ######################################################################################################################################################################################### | 100% 
babel-2.9.0          | 6.2 MB    | ######################################################################################################################################################################################### | 100% 
libssh2-1.9.0        | 225 KB    | ######################################################################################################################################################################################### | 100% 
xorg-libxext-1.3.4   | 54 KB     | ######################################################################################################################################################################################### | 100% 
pandocfilters-1.4.2  | 9 KB      | ######################################################################################################################################################################################### | 100% 
nbformat-5.1.2       | 66 KB     | ######################################################################################################################################################################################### | 100% 
python-dateutil-2.8. | 220 KB    | ######################################################################################################################################################################################### | 100% 
_libgcc_mutex-0.1    | 3 KB      | ######################################################################################################################################################################################### | 100% 
tornado-6.1          | 643 KB    | ######################################################################################################################################################################################### | 100% 
_openmp_mutex-4.5    | 22 KB     | ######################################################################################################################################################################################### | 100% 
sphinxcontrib-appleh | 28 KB     | ######################################################################################################################################################################################### | 100% 
libgfortran-ng-9.3.0 | 22 KB     | ######################################################################################################################################################################################### | 100% 
dcm2niix-1.0.2018111 | 168 KB    | ######################################################################################################################################################################################### | 100% 
importlib-metadata-3 | 23 KB     | ######################################################################################################################################################################################### | 100% 
prometheus_client-0. | 44 KB     | ######################################################################################################################################################################################### | 100% 
xorg-kbproto-1.0.7   | 27 KB     | ######################################################################################################################################################################################### | 100% 
sphinxcontrib-qthelp | 25 KB     | ######################################################################################################################################################################################### | 100% 
hdf5-1.10.6          | 3.1 MB    | ######################################################################################################################################################################################### | 100% 
xorg-libxau-1.0.9    | 13 KB     | ######################################################################################################################################################################################### | 100% 
wxnatpy-0.3.2        | 19 KB     | ######################################################################################################################################################################################### | 100% 
graphite2-1.3.13     | 102 KB    | ######################################################################################################################################################################################### | 100% 
ipython_genutils-0.2 | 21 KB     | ######################################################################################################################################################################################### | 100% 
markupsafe-1.1.1     | 27 KB     | ######################################################################################################################################################################################### | 100% 
xorg-renderproto-0.1 | 9 KB      | ######################################################################################################################################################################################### | 100% 
pathlib2-2.3.5       | 35 KB     | ######################################################################################################################################################################################### | 100% 
libwebp-base-1.2.0   | 808 KB    | ######################################################################################################################################################################################### | 100% 
sphinxcontrib-serial | 24 KB     | ######################################################################################################################################################################################### | 100% 
pexpect-4.8.0        | 47 KB     | ######################################################################################################################################################################################### | 100% 
libgcc-ng-9.3.0      | 7.8 MB    | ######################################################################################################################################################################################### | 100% 
pyopenssl-20.0.1     | 48 KB     | ######################################################################################################################################################################################### | 100% 
entrypoints-0.3      | 8 KB      | ######################################################################################################################################################################################### | 100% 
pypubsub-4.0.3       | 44 KB     | ######################################################################################################################################################################################### | 100% 
certifi-2020.12.5    | 143 KB    | ######################################################################################################################################################################################### | 100% 
jupyterlab_pygments- | 8 KB      | ######################################################################################################################################################################################### | 100% 
sphinxcontrib-htmlhe | 27 KB     | ######################################################################################################################################################################################### | 100% 
python_abi-3.6       | 4 KB      | ######################################################################################################################################################################################### | 100% 
chardet-4.0.0        | 219 KB    | ######################################################################################################################################################################################### | 100% 
libgomp-9.3.0        | 376 KB    | ######################################################################################################################################################################################### | 100% 
xorg-libxdmcp-1.1.3  | 19 KB     | ######################################################################################################################################################################################### | 100% 
matplotlib-base-3.3. | 6.8 MB    | ######################################################################################################################################################################################### | 100% 
setuptools-49.6.0    | 936 KB    | ######################################################################################################################################################################################### | 100% 
python-3.6.13        | 38.4 MB   | ######################################################################################################################################################################################### | 100% 
kiwisolver-1.3.1     | 78 KB     | ######################################################################################################################################################################################### | 100% 
nbclient-0.5.3       | 67 KB     | ######################################################################################################################################################################################### | 100% 
docutils-0.16        | 738 KB    | ######################################################################################################################################################################################### | 100% 
notebook-6.2.0       | 6.2 MB    | ######################################################################################################################################################################################### | 100% 
sphinx-3.5.1         | 1.5 MB    | ######################################################################################################################################################################################### | 100% 
colorama-0.4.4       | 18 KB     | ######################################################################################################################################################################################### | 100% 
libedit-3.1.20191231 | 121 KB    | ######################################################################################################################################################################################### | 100% 
xz-5.2.5             | 343 KB    | ######################################################################################################################################################################################### | 100% 
pycparser-2.20       | 94 KB     | ######################################################################################################################################################################################### | 100% 
xorg-libice-1.0.10   | 58 KB     | ######################################################################################################################################################################################### | 100% 
h5py-3.1.0           | 1.2 MB    | ######################################################################################################################################################################################### | 100% 
ncurses-6.2          | 985 KB    | ######################################################################################################################################################################################### | 100% 
pytz-2021.1          | 239 KB    | ######################################################################################################################################################################################### | 100% 
cryptography-3.4.4   | 631 KB    | ######################################################################################################################################################################################### | 100% 
packaging-20.9       | 35 KB     | ######################################################################################################################################################################################### | 100% 
jupyter_core-4.7.1   | 72 KB     | ######################################################################################################################################################################################### | 100% 
nibabel-3.2.1        | 2.7 MB    | ######################################################################################################################################################################################### | 100% 
webencodings-0.5.1   | 12 KB     | ######################################################################################################################################################################################### | 100% 
traitlets-4.3.3      | 133 KB    | ######################################################################################################################################################################################### | 100% 
jinja2-2.11.3        | 93 KB     | ######################################################################################################################################################################################### | 100% 
testpath-0.4.4       | 85 KB     | ######################################################################################################################################################################################### | 100% 
libblas-3.9.0        | 11 KB     | ######################################################################################################################################################################################### | 100% 
libnghttp2-1.43.0    | 808 KB    | ######################################################################################################################################################################################### | 100% 
xorg-xproto-7.0.31   | 73 KB     | ######################################################################################################################################################################################### | 100% 
sphinx_rtd_theme-0.5 | 2.7 MB    | ######################################################################################################################################################################################### | 100% 
scipy-1.5.3          | 19.1 MB   | ######################################################################################################################################################################################### | 100% 
fslpy-3.5.3          | 3.6 MB    | ######################################################################################################################################################################################### | 100% 
libsodium-1.0.18     | 366 KB    | ######################################################################################################################################################################################### | 100% 
xnat-0.3.25          | 48 KB     | ######################################################################################################################################################################################### | 100% 
send2trash-1.5.0     | 12 KB     | ######################################################################################################################################################################################### | 100% 
jedi-0.17.2          | 957 KB    | ######################################################################################################################################################################################### | 100% 
ipykernel-5.5.0      | 166 KB    | ######################################################################################################################################################################################### | 100% 
argon2-cffi-20.1.0   | 47 KB     | ######################################################################################################################################################################################### | 100% 
liblapack-3.9.0      | 11 KB     | ######################################################################################################################################################################################### | 100% 
sphinxcontrib-devhel | 22 KB     | ######################################################################################################################################################################################### | 100% 
icu-68.1             | 13.0 MB   | ######################################################################################################################################################################################### | 100% 
lcms2-2.12           | 443 KB    | ######################################################################################################################################################################################### | 100% 
jpeg-9d              | 264 KB    | ######################################################################################################################################################################################### | 100% 
progressbar2-3.53.1  | 25 KB     | ######################################################################################################################################################################################### | 100% 
libxml2-2.9.10       | 1.3 MB    | ######################################################################################################################################################################################### | 100% 
isodate-0.6.0        | 25 KB     | ######################################################################################################################################################################################### | 100% 
wheel-0.36.2         | 31 KB     | ######################################################################################################################################################################################### | 100% 
brotlipy-0.7.0       | 341 KB    | ######################################################################################################################################################################################### | 100% 
xorg-libx11-1.7.0    | 923 KB    | ######################################################################################################################################################################################### | 100% 
sphinxcontrib-jsmath | 7 KB      | ######################################################################################################################################################################################### | 100% 
pthread-stubs-0.4    | 5 KB      | ######################################################################################################################################################################################### | 100% 
libuuid-2.32.1       | 28 KB     | ######################################################################################################################################################################################### | 100% 
pickleshare-0.7.5    | 9 KB      | ######################################################################################################################################################################################### | 100% 
glib-2.66.7          | 443 KB    | ######################################################################################################################################################################################### | 100% 
nbconvert-6.0.7      | 533 KB    | ######################################################################################################################################################################################### | 100% 
libstdcxx-ng-9.3.0   | 4.0 MB    | ######################################################################################################################################################################################### | 100% 
async_generator-1.10 | 18 KB     | ######################################################################################################################################################################################### | 100% 
libev-4.33           | 104 KB    | ######################################################################################################################################################################################### | 100% 
fsleyes-0.32.3       | 28.2 MB   | ######################################################################################################################################################################################### | 100% 
wcwidth-0.2.5        | 33 KB     | ######################################################################################################################################################################################### | 100% 
libglu-9.0.0         | 413 KB    | ######################################################################################################################################################################################### | 100% 
cached-property-1.5. | 10 KB     | ######################################################################################################################################################################################### | 100% 
glib-tools-2.66.7    | 85 KB     | ######################################################################################################################################################################################### | 100% 
atk-1.0-2.36.0       | 560 KB    | ######################################################################################################################################################################################### | 100% 
pip-21.0.1           | 1.1 MB    | ######################################################################################################################################################################################### | 100% 
urllib3-1.26.3       | 99 KB     | ######################################################################################################################################################################################### | 100% 
ptyprocess-0.7.0     | 16 KB     | ######################################################################################################################################################################################### | 100% 
alabaster-0.7.12     | 15 KB     | ######################################################################################################################################################################################### | 100% 
ipython-7.16.1       | 1.1 MB    | ######################################################################################################################################################################################### | 100% 
gtk2-2.24.33         | 7.3 MB    | ######################################################################################################################################################################################### | 100% 
pandoc-2.11.4        | 17.9 MB   | ######################################################################################################################################################################################### | 100% 
fribidi-1.0.10       | 112 KB    | ######################################################################################################################################################################################### | 100% 
pydicom-2.1.2        | 1.3 MB    | ######################################################################################################################################################################################### | 100% 
typing_extensions-3. | 25 KB     | ######################################################################################################################################################################################### | 100% 
mistune-0.8.4        | 54 KB     | ######################################################################################################################################################################################### | 100% 
libxcb-1.13          | 395 KB    | ######################################################################################################################################################################################### | 100% 
parso-0.7.1          | 70 KB     | ######################################################################################################################################################################################### | 100% 
wxpython-4.1.1       | 21.3 MB   | ######################################################################################################################################################################################### | 100% 
attrs-20.3.0         | 41 KB     | ######################################################################################################################################################################################### | 100% 
c-ares-1.17.1        | 111 KB    | ######################################################################################################################################################################################### | 100% 
backcall-0.2.0       | 13 KB     | ######################################################################################################################################################################################### | 100% 
xorg-libxrender-0.9. | 32 KB     | ######################################################################################################################################################################################### | 100% 
defusedxml-0.6.0     | 22 KB     | ######################################################################################################################################################################################### | 100% 
jupyter_client-6.1.1 | 76 KB     | ######################################################################################################################################################################################### | 100% 
libiconv-1.16        | 1.4 MB    | ######################################################################################################################################################################################### | 100% 
pcre-8.44            | 261 KB    | ######################################################################################################################################################################################### | 100% 
trimesh-3.9.8        | 504 KB    | ######################################################################################################################################################################################### | 100% 
imagesize-1.2.0      | 8 KB      | ######################################################################################################################################################################################### | 100% 
gstreamer-1.18.3     | 2.0 MB    | ######################################################################################################################################################################################### | 100% 
zlib-1.2.11          | 106 KB    | ######################################################################################################################################################################################### | 100% 
nest-asyncio-1.4.3   | 9 KB      | ######################################################################################################################################################################################### | 100% 
six-1.15.0           | 14 KB     | ######################################################################################################################################################################################### | 100% 
numpy-1.19.5         | 5.3 MB    | ######################################################################################################################################################################################### | 100% 
freeglut-3.0.0       | 247 KB    | ######################################################################################################################################################################################### | 100% 
freetype-2.10.4      | 890 KB    | ######################################################################################################################################################################################### | 100% 
libcblas-3.9.0       | 11 KB     | ######################################################################################################################################################################################### | 100% 
rtree-0.9.7          | 45 KB     | ######################################################################################################################################################################################### | 100% 
fontconfig-2.13.1    | 344 KB    | ######################################################################################################################################################################################### | 100% 
openssl-1.1.1j       | 2.1 MB    | ######################################################################################################################################################################################### | 100% 
snowballstemmer-2.1. | 57 KB     | ######################################################################################################################################################################################### | 100% 
xorg-xextproto-7.3.0 | 28 KB     | ######################################################################################################################################################################################### | 100% 
readline-8.0         | 281 KB    | ######################################################################################################################################################################################### | 100% 
backports-1.0        | 4 KB      | ######################################################################################################################################################################################### | 100% 
ld_impl_linux-64-2.3 | 618 KB    | ######################################################################################################################################################################################### | 100% 
zipp-3.4.0           | 11 KB     | ######################################################################################################################################################################################### | 100% 
backports.functools_ | 8 KB      | ######################################################################################################################################################################################### | 100% 
libglib-2.66.7       | 3.0 MB    | ######################################################################################################################################################################################### | 100% 
fsleyes-widgets-0.11 | 505 KB    | ######################################################################################################################################################################################### | 100% 
ca-certificates-2020 | 137 KB    | ######################################################################################################################################################################################### | 100% 
zeromq-4.3.4         | 352 KB    | ######################################################################################################################################################################################### | 100% 
expat-2.2.10         | 164 KB    | ######################################################################################################################################################################################### | 100% 
pixman-0.40.0        | 627 KB    | ######################################################################################################################################################################################### | 100% 
pyrsistent-0.17.3    | 89 KB     | ######################################################################################################################################################################################### | 100% 
libcurl-7.71.1       | 312 KB    | ######################################################################################################################################################################################### | 100% 
pango-1.42.4         | 533 KB    | ######################################################################################################################################################################################### | 100% 
olefile-0.46         | 32 KB     | ######################################################################################################################################################################################### | 100% 
dataclasses-0.7      | 21 KB     | ######################################################################################################################################################################################### | 100% 
zstd-1.4.9           | 431 KB    | ######################################################################################################################################################################################### | 100% 
tk-8.6.10            | 3.2 MB    | ######################################################################################################################################################################################### | 100% 
pyopengl-3.1.5       | 854 KB    | ######################################################################################################################################################################################### | 100% 
libffi-3.3           | 51 KB     | ######################################################################################################################################################################################### | 100% 
gst-plugins-base-1.1 | 2.5 MB    | ######################################################################################################################################################################################### | 100% 
indexed_gzip-1.4.0   | 523 KB    | ######################################################################################################################################################################################### | 100% 
requests-2.25.1      | 51 KB     | ######################################################################################################################################################################################### | 100% 
bleach-3.3.0         | 111 KB    | ######################################################################################################################################################################################### | 100% 
libtiff-4.2.0        | 633 KB    | ######################################################################################################################################################################################### | 100% 
pyparsing-2.4.7      | 60 KB     | ######################################################################################################################################################################################### | 100% 
libspatialindex-1.9. | 4.6 MB    | ######################################################################################################################################################################################### | 100% 
libopenblas-0.3.12   | 8.9 MB    | ######################################################################################################################################################################################### | 100% 
cffi-1.14.5          | 225 KB    | ######################################################################################################################################################################################### | 100% 
libpng-1.6.37        | 306 KB    | ######################################################################################################################################################################################### | 100% 
cycler-0.10.0        | 9 KB      | ######################################################################################################################################################################################### | 100% 
pillow-8.1.1         | 679 KB    | ######################################################################################################################################################################################### | 100% 
terminado-0.9.2      | 26 KB     | ######################################################################################################################################################################################### | 100% 
gdk-pixbuf-2.42.2    | 611 KB    | ######################################################################################################################################################################################### | 100% 
idna-2.10            | 52 KB     | ######################################################################################################################################################################################### | 100% 
pyzmq-22.0.3         | 496 KB    | ######################################################################################################################################################################################### | 100% 
gettext-0.19.8.1     | 3.6 MB    | ######################################################################################################################################################################################### | 100% 
pygments-2.8.0       | 736 KB    | ######################################################################################################################################################################################### | 100% 
libgfortran5-9.3.0   | 2.0 MB    | ######################################################################################################################################################################################### | 100% 
python-utils-2.5.5   | 15 KB     | ######################################################################################################################################################################################### | 100% 
lz4-c-1.9.3          | 179 KB    | ######################################################################################################################################################################################### | 100% 
xorg-libsm-1.2.3     | 25 KB     | ######################################################################################################################################################################################### | 100% 
jsonschema-3.2.0     | 45 KB     | ######################################################################################################################################################################################### | 100% 
harfbuzz-2.7.4       | 1.9 MB    | ######################################################################################################################################################################################### | 100% 
sqlite-3.34.0        | 1.4 MB    | ######################################################################################################################################################################################### | 100% 
Preparing transaction: done
Verifying transaction: done
Executing transaction: -                                                                                                                                                                                                          done
```
</details>

This install log shows that several Python and non-Python packages are updated/modified, which is concerning. I don't believe we are currently testing the effect that this has on the base SCT functionality.

Also, here is the diff between `sct_check_dependencies`: [(before/after)](https://www.diffchecker.com/Z5YZDJKT). 

The main concern is `h5py`. We pin `h5py==2.1.0` for the reasons in #2987. But, it gets updated because `conda install` isn't aware of our `requirements.txt` file (where `h5py` is pinned). So, the issue from #2987 returns:

```python
AttributeError: 'str' object has no attribute 'decode'
```

**Comment by joshuacwnewton** [comment]
A [user on the forum](https://forum.spinalcordmri.org/t/how-to-open-sct/647/4?u=joshuacwnewton) is reporting the same `h5py` error described above on WSL. 

I don't have confirmation yet, but I suspect the user installed FSLeyes while following along with the SCT course. 

If true, then coming up with better FSLeyes install instructions should be done ASAP (i.e. as part of the 5.2.1 patch release.)

**Comment by jcohenadad** [comment]
> coming up with better FSLeyes install instructions should be done ASAP 

I completely agree with this. FSLeyes installation has been a nightmare for other projects are well (shimming-toolbox and axondeepseg). I increasingly think we should isolate FSeyes install from our package install. The good thing with SCT is that we _can_ do that because we have launchers that call our the proper venv. In other packages, we are even considering creating launchers to precisely remove the venv management burden on the user side, see e.g. https://github.com/shimming-toolbox/shimming-toolbox/issues/237

**Comment by joshuacwnewton** [comment]
> A [user on the forum](https://forum.spinalcordmri.org/t/how-to-open-sct/647/4?u=joshuacwnewton) is reporting the same `h5py` error described above on WSL.

The user has since been able to proceed without issue, so I'm removing this from the next release milestone.

But, I'd still like to look at this after the next release is completed.

**Comment by claragiroux** [comment]
I also had this issue right after installing FSLeyes on WSL with Ubuntu 20.04. For example when I was running sct_testing I get the following log:

<details>
<summary> Terminal output: </summary>
clara@DESKTOP-8DII8HA:~/sct$ sct_testing

--
Spinal Cord Toolbox (git-3242-QC-dmri-fmri-moco-9e53dc6f906d65a1193c7e05dc6aff396277461f*)

sct_testing
--

Creating temporary folder (/tmp/sct-20210413191601.899659-0gk6vk9a)
Will run through the following tests:
- sequentially: sct_deepseg_gm sct_deepseg_lesion sct_deepseg_sc
- in parallel with 8 jobs: sct_analyze_lesion sct_analyze_texture sct_apply_transfo sct_convert sct_compute_ernst_angle sct_compute_hausdorff_distance sct_compute_mtr sct_compute_mscc sct_compute_snr sct_concat_transfo sct_create_mask sct_crop_image sct_dice_coefficient sct_detect_pmj sct_dmri_compute_dti sct_dmri_concat_b0_and_dwi sct_dmri_concat_bvals sct_dmri_concat_bvecs sct_dmri_compute_bvalue sct_dmri_moco sct_dmri_separate_b0_and_dwi sct_dmri_transpose_bvecs sct_extract_metric sct_flatten_sagittal sct_fmri_compute_tsnr sct_fmri_moco sct_get_centerline sct_image sct_label_utils sct_label_vertebrae sct_maths sct_merge_images sct_process_segmentation sct_propseg sct_qc sct_register_multimodal sct_register_to_template sct_resample sct_smooth_spinalcord sct_straighten_spinalcord sct_warp_template
Checking sct_deepseg_gm.............................[FAIL]

   ====================================================================================================
   sct_deepseg_gm -i t2s/t2s_uncropped.nii.gz -o output.nii.gz -qc testing-qc
   ====================================================================================================


   --
   Spinal Cord Toolbox (git-3242-QC-dmri-fmri-moco-9e53dc6f906d65a1193c7e05dc6aff396277461f*)

   sct_deepseg_gm -i t2s/t2s_uncropped.nii.gz -o output.nii.gz -qc testing-qc
   --

   Traceback (most recent call last):
   File "/home/clara/sct/spinalcordtoolbox/scripts/sct_deepseg_gm.py", line 138, in <module>
   main(sys.argv[1:])
   File "/home/clara/sct/spinalcordtoolbox/scripts/sct_deepseg_gm.py", line 121, in main
   use_tta)
   File "/home/clara/sct/spinalcordtoolbox/deepseg_gm/deepseg_gm.py", line 313, in segment_file
   use_tta)
   File "/home/clara/sct/spinalcordtoolbox/deepseg_gm/deepseg_gm.py", line 233, in segment_volume
   deepgmseg_model.load_weights(model_abs_path)
   File "/home/clara/sct/python/envs/venv_sct/lib/python3.6/site-packages/keras/engine/topology.py", line 2656, in load_weights
   f, self.layers, reshape=reshape)
   File "/home/clara/sct/python/envs/venv_sct/lib/python3.6/site-packages/keras/engine/topology.py", line 3328, in load_weights_from_hdf5_group
   original_keras_version = f.attrs['keras_version'].decode('utf8')
   AttributeError: 'str' object has no attribute 'decode'
Checking sct_deepseg_lesion.........................[FAIL]

   ====================================================================================================
   sct_deepseg_lesion -i t2/t2.nii.gz -c t2
   ====================================================================================================


   --
   Spinal Cord Toolbox (git-3242-QC-dmri-fmri-moco-9e53dc6f906d65a1193c7e05dc6aff396277461f*)

   sct_deepseg_lesion -i t2/t2.nii.gz -c t2
   --


   Method:
   Centerline algorithm: svm
   Assumes brain section included in the image: True

   Creating temporary folder (/tmp/sct-20210413191612.063225-gklqrlcx)
   Reorient the image to RPI, if necessary...

   Finding the spinal cord centerline...
   Creating temporary folder (/tmp/sct-20210413191612.405158-yerrsx40)
   Remove temporary files...
   rm -rf /tmp/sct-20210413191612.405158-yerrsx40

   Cropping the image around the spinal cord...
   Normalizing the intensity...
   load data...

   Segmenting the MS lesions using deep learning on 3D patches...
   Traceback (most recent call last):
   File "/home/clara/sct/spinalcordtoolbox/scripts/sct_deepseg_lesion.py", line 173, in <module>
   main(sys.argv[1:])
   File "/home/clara/sct/spinalcordtoolbox/scripts/sct_deepseg_lesion.py", line 149, in main
   brain_bool=brain_bool, remove_temp_files=remove_temp_files, verbose=verbose)
   File "/home/clara/sct/spinalcordtoolbox/deepseg_lesion/core.py", line 201, in deep_segmentation_MSlesion
   im=im_res3d.copy())
   File "/home/clara/sct/spinalcordtoolbox/deepseg_lesion/core.py", line 88, in segment_3d
   seg_model = load_trained_model(model_fname)
   File "/home/clara/sct/spinalcordtoolbox/deepseg_sc/cnn_models_3d.py", line 91, in load_trained_model
   return load_model(model_file, custom_objects=custom_objects)
   File "/home/clara/sct/python/envs/venv_sct/lib/python3.6/site-packages/keras/models.py", line 242, in load_model
   model_config = json.loads(model_config.decode('utf-8'))
   AttributeError: 'str' object has no attribute 'decode'
Checking sct_deepseg_sc.............................[FAIL]

   ====================================================================================================
   sct_deepseg_sc -i t2/t2.nii.gz -c t2 -qc testing-qc
   ====================================================================================================


   --
   Spinal Cord Toolbox (git-3242-QC-dmri-fmri-moco-9e53dc6f906d65a1193c7e05dc6aff396277461f*)

   sct_deepseg_sc -i t2/t2.nii.gz -c t2 -qc testing-qc
   --

   Config deepseg_sc:
   Centerline algorithm: svm
   Brain in image: True
   Kernel dimension: 2d
   Contrast: t2
   Threshold: 0.7
   Creating temporary folder (/tmp/sct-20210413191620.212629-rb8enj1x)
   Reorient the image to RPI, if necessary...
   Finding the spinal cord centerline...
   Creating temporary folder (/tmp/sct-20210413191620.745062-96jwq44p)
   Remove temporary files...
   rm -rf /tmp/sct-20210413191620.745062-96jwq44p
   Cropping the image around the spinal cord...
   Normalizing the intensity...
   Segmenting the spinal cord using deep learning on 2D patches...
   Traceback (most recent call last):
   File "/home/clara/sct/spinalcordtoolbox/scripts/sct_deepseg_sc.py", line 214, in <module>
   main(sys.argv[1:])
   File "/home/clara/sct/spinalcordtoolbox/scripts/sct_deepseg_sc.py", line 199, in main
   threshold_seg=threshold, remove_temp_files=remove_temp_files, verbose=verbose)
   File "/home/clara/sct/spinalcordtoolbox/deepseg_sc/core.py", line 506, in deep_segmentation_spinalcord
   im_in=im_norm_in)
   File "/home/clara/sct/spinalcordtoolbox/deepseg_sc/core.py", line 347, in segment_2d
   seg_model.load_weights(model_fname)
   File "/home/clara/sct/python/envs/venv_sct/lib/python3.6/site-packages/keras/engine/topology.py", line 2656, in load_weights
   f, self.layers, reshape=reshape)
   File "/home/clara/sct/python/envs/venv_sct/lib/python3.6/site-packages/keras/engine/topology.py", line 3328, in load_weights_from_hdf5_group
   original_keras_version = f.attrs['keras_version'].decode('utf8')
   AttributeError: 'str' object has no attribute 'decode'
Checking sct_analyze_lesion.........................[OK]
Checking sct_analyze_texture........................[OK]
Checking sct_apply_transfo..........................[OK]
Checking sct_convert................................[OK]
Checking sct_compute_ernst_angle....................[OK]
Checking sct_compute_hausdorff_distance.............[OK]
Checking sct_compute_mtr............................[OK]
Checking sct_compute_mscc...........................[OK]
Checking sct_compute_snr............................[OK]
Checking sct_concat_transfo.........................[OK]
Checking sct_create_mask............................[OK]
Checking sct_crop_image.............................[OK]
Checking sct_dice_coefficient.......................[OK]
Checking sct_detect_pmj.............................[OK]
Checking sct_dmri_compute_dti.......................[OK]
Checking sct_dmri_concat_b0_and_dwi.................[OK]
Checking sct_dmri_concat_bvals......................[OK]
Checking sct_dmri_concat_bvecs......................[OK]
Checking sct_dmri_compute_bvalue....................[OK]
Checking sct_dmri_moco..............................[OK]
Checking sct_dmri_separate_b0_and_dwi...............[OK]
Checking sct_dmri_transpose_bvecs...................[OK]
Checking sct_extract_metric.........................[OK]
Checking sct_flatten_sagittal.......................[OK]
Checking sct_fmri_compute_tsnr......................[OK]
Checking sct_fmri_moco..............................[OK]
Checking sct_get_centerline.........................[OK]
Checking sct_image..................................[OK]
Checking sct_label_utils............................[OK]
Checking sct_label_vertebrae........................[OK]
Checking sct_maths..................................[OK]
Checking sct_merge_images...........................[OK]
Checking sct_process_segmentation...................[OK]
Checking sct_propseg................................[OK]
Checking sct_qc.....................................[OK]
Checking sct_register_multimodal....................[OK]
Checking sct_register_to_template...................[OK]
Checking sct_resample...............................[OK]
Checking sct_smooth_spinalcord......................[OK]
Checking sct_straighten_spinalcord..................[OK]
Checking sct_warp_template..........................[OK]
status: [1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
Failures: sct_deepseg_gm sct_deepseg_lesion sct_deepseg_sc
Finished! Elapsed time: 224s
</details>

**Comment by joshuacwnewton** [comment]
Thank you for the reminder about this issue!

> Is there a quick way to change the current h5py version? I didn't find anything that worked.

Just to double-check what you've tried, have you tried uninstalling `h5py` then reinstalling the older version? e.g.:

```
source ${SCT_DIR}/python/etc/profile.d/conda.sh
conda activate venv_sct
conda remove h5py
conda install h5py=2.10.0
```

**Comment by claragiroux** [comment]
> source ${SCT_DIR}/python/etc/profile.d/conda.sh
> conda activate venv_sct
> conda remove h5py
> conda install h5py=2.10.0

Thanks that worked! I was trying with pip install instead of conda... 

I also had to reinstall pytz and dateutil right after!
